### PR TITLE
Add _contains_html function

### LINKED ISSUE #3: Create `contains_html` function
The `send_email` function checks if the body contains html and sets the `subtype` accordingly. Export this functionality into a function called `contains_html` and write tests for the function.

### PR DESCRIPTION
**Pull request by michael-bentz**
**What does this change do?** 

Adds `_contains_html` function with unit tests.

**Why was this change made?** 

To close #3.

## Verification

1. Verify the logic of the test_contains_html unit test
2. Verify that all automated workflows pass

- Verify the thing does this: ...
- Verify the thing does not do that: ...

## Affirmations

All of these should have a check by them. Any exception requires an explanation.

* [x] I have added docstrings with details on keyword arguments to new functions following the convention detailed in this [gist](https://gist.github.com/mbentz-uf/0433c32f260a0a06f57a9ff32fcef252)
* [x] I have added type hinting to new function parameters.
* [x] I have added tests to new functions, following the rules of [F.I.R.S.T.](https://wiki.ctsi.ufl.edu/books/testing/page/how-to-write-python-unit-tests).
* [x] I matched the style of the existing code.
* [x] I added and updated any relevant documentation (inline, `README`, `CHANGELOG`, and such).
* [x] I used Python's type hinting.
* [x] I ran the automated tests and ensured they **ALL** passed.
* [x] I ran the linter and ensured my changes have not introduced **ANY** warnings or errors.
* [x] I have made an effort to minimize code changes and have not included any cruft (preference files, *.pyc files, old comments, print-debugging, unused variables).
* [x] I have made an effort maintain a clear commit history (haven't merged other branches or rebased improperly).
* [x] I have written the code myself or have given credit where credit is due.

### CODE REVIEW

File: tests/py_utils/test_utils.py [hunk 1]
```diff
@@ -9,6 +9,13 @@ class TestUtils(unittest.TestCase):
     first_duplicate_file_name = 'get_unique_filename_test (1).file'
     second_duplicate_file_name = 'get_unique_filename_test (2).file'
 
+    def test_contains_html(self):
+        html_markup = "<html><p></p></html>"
+        non_html_text = "This is a sentence with special characters, < ? ! . '"
+
+        self.assertEquals(True, utils._contains_html(html_markup))
```
Comment (mehuljhaver4) [inline tests/py_utils/test_utils.py:16]: line 16 and similar lines show the following problem:

`Cannot access member "assertEquals" for type "TestUtils"
 Member "assertEquals" is unknownPylanc` [reportGeneralTypeIssues](https://github.com/microsoft/pyright/blob/main/docs/configuration.md#reportGeneralTypeIssues)

But the tests run successfully

**Review by mehuljhaver4 — APPROVED** [review]
* Verified the logic of the test_contains_html unit test
* Verified that all automated workflows passed

File: tests/py_utils/test_utils.py (hunk 1 above)
Comment (michael-bentz) [reply tests/py_utils/test_utils.py:16]: I'm not sure how you're getting this. I don't see it locally and the lint workflow is passing.

Comment (mehuljhaver4) [reply tests/py_utils/test_utils.py:16]: Must be some issue at my end. I'll merge it.

**Pull request merged**